### PR TITLE
CoalaBear: Change ``coala`` to ``Coala``

### PR DESCRIPTION
**Pull request by Asnelchristian**
``Coala`` is always written with an upper case ``C``.

### CODE REVIEW

File: bears/general/CoalaBear.py [hunk 1]
```diff
@@ -16,18 +16,18 @@ class coalaBear(LocalBear):
...
+            wrong_spelling = r'c([oO][aA][lL][aA])'
             corrected += [re.sub(wrong_spelling,
-                                 lambda match: 'c' + match.group(1),
+                                 lambda match: 'C' + match.group(1),
                                  line)]
         diffs = Diff.from_string_arrays(file, corrected).split_diff()
         for diff in diffs:
             yield Result(self,
-                         "``coala`` is always written with a lower case ``c``",
+                         "``Coala`` is always written with an upper case ``C``",
```
Comment (sils1297) [inline bears/general/CoalaBear.py]: NOOOOOOOOOOOOOOOO!

**Comment by Adrianzatreanu** [comment]
ack be89699

**Comment by sils1297** [comment]
unack be89699
<https://github.com/coala-analyzer/coala-bears/commit/be8969989096a42f15c04e9614da7422a4d15c07>

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala-analyzer.org/ - http://viperdev.io/ - http://gitmate.com/

File: bears/general/CoalaBear.py (hunk 1 above)
Comment (Asnelchristian) [inline bears/general/CoalaBear.py]: WHY ?

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala-bears/pull/668/commits/c6c416505b6d5bcd7d5bd7778f3fd0347daf82ee needs work

**Comment by Makman2** [comment]
wait isn't it always written with a lower case 'c'? Did I miss the memo? :O

**Comment by Asnelchristian** [comment]
It seems it was just some sort of joke :) @Makman2

**Comment by Adrianzatreanu** [comment]
:(

**Comment by Makman2** [comment]
:3

**Pull request closed without merging**